chardet/utf8prober.py

```diff
@@ -75,7 +75,11 @@ def feed(self, byte_str):
 
     def get_confidence(self):
         unlike = 0.99
-        if self._num_mb_chars < 6:
+        if self._num_mb_chars == 1:
+            # guarantee to be prefered over ISO-8859-1, defaults to 0.731
```

Comment:
```suggestion
            # guaranteed to be preferred over ISO-8859-1, evaluates to 0.731
```

Reply:
thank you @raylu i changed it accordingly.

Reply:
(to be clear, I'm not a maintainer on this project. I hope this gets merged, though! we're currently monkeypatching this behavior in)